locales/en.yml

```diff
@@ -75,7 +75,9 @@ en:
 
           Service:
               A service provided by sccli or systemd. For example:
-               docker, k8s, or openshift etc.
+               sccli service like docker, kubernetes, openshift etc
+               or
+               systemd service like libvirtd etc
```

Comment:
@hferentschik It was been discuss when we are implementing `restart` feature. If you see the [code implementation](https://github.com/projectatomic/vagrant-service-manager/blob/master/lib/vagrant-service-manager/command.rb#L185), we are supporting `systemd` service through CLI.

cc @bexelbie